site/app/controllers/admin/UsersController.php

```diff
@@ -355,7 +355,7 @@ public function updateUser($type = 'users') {
             $this->core->addSuccessMessage("User '{$user->getId()}' updated");
         }
         else {
-            if ($this->core->getQueries()->getSubmittyUser($_POST['user_id']) === null) {
+            if (($u = $this->core->getQueries()->getSubmittyUser($_POST['user_id'])) === null) {
```

Comment:
This syntax is somewhat confusing, especially given that `$u` is used again 368.  It might be more clear to do the variable assignment on the line above, and then only perform the check on this line.